airbyte-webapp/src/components/BarChart/BarChart.tsx

```diff
@@ -16,6 +16,11 @@ const BarChart: React.FC<BarChartProps> = ({ data, legendLabels, xLabel, yLabel
   const chartLinesColor = theme.greyColor20;
   const chartTicksColor = theme.lightTextColor;
 
+  const width = Math.min(
+    Math.max([...data].sort((a, b) => b.value - a.value)[0].value.toFixed(0).length * 10, 80),
```

Comment:
Assumes that there's always at least 1 item in the array?

Reply:
The default data is an array of dates + values at 0.